next.config.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-  generateBuildId: async () => "001",
-
+  generateBuildId: async () => '001',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7096303)
```suggestion
  generateBuildId: async () => "001",
```